elasticsearch/templates/statefulset.yaml

```diff
@@ -216,3 +216,50 @@ spec:
         {{- if .Values.extraVolumeMounts }}
 {{ tpl .Values.extraVolumeMounts . | indent 10 }}
         {{- end }}
+      {{- if eq .Values.roles.master "true" }}
```

Comment:
Could you include a link to https://github.com/elastic/helm-charts/issues/63 here. Just so future people looking at this won't be super confused what it is and why it exists.